packages/neos-ui-editors/src/Editors/AssetEditor/index.js

```diff
@@ -140,6 +160,7 @@ export default class AssetEditor extends PureComponent {
     }
 
     handleMediaSelected = assetIdentifier => {
+        console.log('handleMediaSelected', assetIdentifier, this.props);
```

Comment:
I guess this console log needs to be removed

Reply:
Yes indeed.